### PR TITLE
Brig map changes

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Expanded forensic lab and hospital a little, ~gave it a medicine vendor.~
Lethal injection closet moved to execution room. See screenshots for more details.
<details>
<summary>
Before
</summary>

image

image

</details>

<details>
<summary>
After
</summary>

image

image

</details>

<details>
<summary>
Vendor
</summary>

Regular:
image

Hacked:
image

</details>

UPD: Vendor have been removed.

## Why It's Good For The Game

More use for money for IH and actual source of chems when Moebius Medical not running chemistry (90% of the time), even if at robbing price.

## Changelog
:cl:
add: expanded hospital a little
/:cl:

### CODE REVIEW

**Comment by CeUvi** [comment]
chemist removal PR when?

**Comment by Omni44** [comment]
How far down the agenda list is replacing entire eris with IH&IH related content? I seriously want to know.

**Comment by CeUvi** [comment]
actually if brigmed had more medkits it could replace moemed completely lmao.

**Comment by SirRichardFrancis** [comment]
> How far down the agenda list is replacing entire eris with IH&IH related content? I seriously want to know.

More like with NT&IH.

**Comment by Omni44** [comment]
NT udpate at least followed some pattern, there was design document made by Genesse that was somewhat well structured, had clear vision of what it wants to achieve and what NT is supposed to be. What came after Genesse varies wildly in quality, But besides that OG Nt update 2 was sensible (and needed).
You people are just steadily replacing everything with IH content (You turning rare, niche but cool item MIU was into what's essentially IH being able to play as AI not to look very far) /fill any niche they weren't able to cover so far (quadrillion of guns from get go no matter whether they fit or not), from top to bottom and shit out buffs, literally on weekly basis. All with next to 0 supervision or even any framework of some, I dunno, mechanic so that IH has to work for anything to actually earn the goodies? Like most other departments do? 
Coming from used-to-be one of the most notorious IH mains? You guys are insane.

**Comment by SirRichardFrancis** [comment]
>Coming from used-to-be one of the most notorious IH mains

image

**Comment by MLGTASTICa** [comment]
Good intention at heart , but this simply can't be done. The world is not fair and antagonists are indeed  meant to have the advantage agaisnt IH on the medical frontier.

**Comment by TheShown911** [comment]
Surprise surprise. Yet ANOTHER PR with IH getting buffed. It's only been the fiftieth time this week. Ever notice how IH ALWAYS gets buffed either indirectly, or ALWAYS wins the cool new gun diceroll? You think this is a coincidence? I think not, especially with allegations that (former and current) staff developers have accepted playing IH to have fun (powergame). Do you think a person like IH players wouldn't alter their own gear to ALWAYS WIN FIGHTS? I don't think so. Admins? I leave the case in your hands

**Comment by CeUvi** [comment]
uh oh kirov gets gitbanned and gamebanned

**Comment by ValoTheValo** [comment]
I like the new design but why does IH get a chem machine. they should need to get chems from moebius and such.

**Comment by Wouju** [comment]
I kinda like the forsenics office/room change but giving IH a fucking chem machine? why would they need it??

You should try and interact with medbay, hell IH already get their own mini medbay so giving them their own chemistry makes no fucking senses

this PR makes me puke

**Comment by GrayRachnid** [comment]
Give IH a vendor where they can spend their 'fines' money on instead to buy chems in pill bottles or injectors rather than make them print them out for free. This'll encourage IH to actually punish people and fine them instead of going for kills every time.

**Comment by thedarkcrusader** [comment]
> Give IH a vendor where they can spend their 'fines' money on instead to buy chems in pill bottles or injectors rather than make them print them out for free. This'll encourage IH to actually punish people and fine them instead of going for kills every time.



> Give IH a vendor where they can spend their 'fines' money on instead to buy chems in pill bottles or injectors rather than make them print them out for free. This'll encourage IH to actually punish people and fine them instead of going for kills every time.

I like this idea i like it a lot

**Comment by Humonitarian** [comment]
Surprise surprise. Yet ANOTHER PR with IH getting buffed. It's only been the fiftieth time this week. Ever notice how IH ALWAYS gets buffed either indirectly, or ALWAYS wins the cool new gun diceroll? You think this is a coincidence? I think not, especially with allegations that (former and current) staff developers have accepted playing IH to have fun (powergame). Do you think a person like IH players wouldn't alter their own gear to ALWAYS WIN FIGHTS? I don't think so. Admins? I leave the case in your hands

**Comment by SirRichardFrancis** [comment]
> Give IH a vendor where they can spend their 'fines' money on instead to buy chems in pill bottles or injectors rather than make them print them out for free. This'll encourage IH to actually punish people and fine them instead of going for kills every time.

Touché.

> this PR makes me puke
> Good intention at heart , but this simply can't be done.
> uh oh kirov gets gitbanned and gamebanned
> they should need to get chems from moebius and such.

How medspecs going on a pilgrimage to chemistry roundstart is almost comical, even more so IH making a ghetto chemistry, a real bruh moment. That _if_ they concerned about not dying the moment gunfight starts. They mostly don't and one ~powergamer~ skilled antagonist slaughters entire department no problem.
As if moe actually care to make and sell chems. Most often than not there is nobody to man chemistry.

**Comment by Kegdo** [comment]
Don't know what you guys are talking about, but DAMN this pr is amazing; medspecs'd be able to actually HEAL, because now they can actually make healing chems instead of waiting ~~forever~~ 6 years for chemists to make them

**Comment by CeUvi** [comment]
removing amobus chemistry as soon as this gets merged

**Comment by ValoTheValo** [comment]
Yeah ironhammer are the main customers for moebius chem and this would make it so ih never goes to chem

**Comment by thedarkcrusader** [comment]
> Yeah ironhammer are the main customers for moebius chem and this would make it so ih never goes to chem

Make a new chem dispenser that only has a limited number of chemicals that can be  dispensed like the industrial one in Moe sci or if you wanna be funny make  it charge IH account every 5 units

**Comment by Kegdo** [comment]
> > Yeah ironhammer are the main customers for moebius chem and this would make it so ih never goes to chem
> 
> Make a new chem dispenser that only has a limited number of chemicals that can be  dispensed like the industrial one in Moe sci or if you wanna be funny make  it charge IH account every 5 units

Some healing chems instead would be enough then, IH doesn't even have any tricord roundtart, making bone fractures a pain. Though, I may have an idea how to fix it without making chems, but I'd need to code it

**Comment by SirRichardFrancis** [comment]
Description have been updated.

**Comment by Kegdo** [comment]
This is a robbery

**Comment by CeUvi** [comment]
now go and get new sprite for that vendor.

**Comment by Voldirs** [comment]
I'd say move contraband medicals to common section because contraband shouldn't be a thing in the first place since these meds are not illegal. We had "contraband" goods in vendors since that time when economy and agreement weren't well established, but now they are. It only adds one step for someone to bring gloves and tools and hack it just to get pills that also cost a lot.

**Comment by Humonitarian** [comment]
IH has a sleeper, that's a reliable supply of Dylovene, Paracetamol and Tricordrazine.

The only really desirable item is spaceacillin, as they don't have access to that.

If we really want IH to have field supplies, stick to dylo and inap so they have to use their pair of brain cells to inject both to get tric.

Contraband having high power items is bad as well in my opinion, consider putting Soporific and Toxin bottles instead.

**Comment by SirRichardFrancis** [comment]
> IH has a sleeper, that's a reliable supply of Dylovene, Paracetamol and Tricordrazine.

Pumping those out is a reliable way of getting free chems, but that's pretty much an exploit.

> The only really desirable item is spaceacillin, as they don't have access to that.
> If we really want IH to have field supplies, stick to dylo and inap so they have to use their pair of brain cells to inject both to get tric.
> Contraband having high power items is bad as well in my opinion, consider putting Soporific and Toxin bottles instead.

"Toxin and sleep toxin" is useless for healing, m'okay, and IHS have no actual need for those.
"High power items" come with a price tag attached and unreasonably high one, if you ask me.
Any contractor can get those "high power items" and much more roundstart no problem, while PMC grunts not supplied with even cheapest chems, even for money.
"Buying from Moe" is not a thing in most rounds for various reasons, main being absence of Moe personnel.
Sure, IH can steal from medbay, but that's kind of stupid, dunno.
Mixing tric from bottles is practical, but autoinjectors look like a thing military would use.

I've been thinking about giving them personal medkits, with like couple of autoinjectors and salve/bandages, but that will raise even more complains, so whatever.

**Comment by SirRichardFrancis** [comment]
> I don't think medspecs are really lacking medical equipment as for now. They've got everything, but stims and painkillers, but these are luxury rather than necessity.

Painkillers _is_ a necessity with our current combat system, like it or not, and they don't really have chems for use in field, only in the slipper, which is not the same at all.

**Comment by Iglpigl** [comment]
> > I don't think medspecs are really lacking medical equipment as for now. They've got everything, but stims and painkillers, but these are luxury rather than necessity.
> 
> Painkillers _is_ a necessity with our current combat system, like it or not, and they don't really have chems for use in field, only in the slipper, which is not the same at all.

Moot point sleeper has painkiller (paracetamol).

**Comment by Kegdo** [comment]
(paracetamol is shit)
(tricord is too)
MedSpec needs at least bicaridine to properly fix broken bones

**Comment by Voldirs** [comment]
They are not necessity for treatment, they are necessity for staying up in fight. And still medspec has everything to just treat operatives, maybe far from instantly, maybe they have to drag the body to their lab, but you know, fast healing chems are luxury. Though inability to fix bones when the body part has over 60 damage should be removed if it isn't yet.

**Comment by Kegdo** [comment]
> They are not necessity for treatment, they are necessity for staying up in fight. And still medspec has everything to just treat operatives, maybe far from instantly, maybe they have to drag the body to their lab, but you know, fast healing chems are luxury. Though inability to fix bones when the body part has over 60 damage should be removed if it isn't yet.

I have an idea how to tweak this a bit, and I'll make it this week I think.

**Comment by SirRichardFrancis** [comment]
> They are not necessity for treatment, they are necessity for staying up in fight. And still medspec has everything to just treat operatives, maybe far from instantly, maybe they have to drag the body to their lab, but you know, fast healing chems are luxury. Though inability to fix bones when the body part has over 60 damage should be removed if it isn't yet.

It's not about medspecs, but IHS in general. Have no painkillers => eat shit and die. Which they routinely do. Sometimes to just one guy who pumped up on meralyne and oxycodone he got roundstart from uplink with zero effort.

**Comment by Voldirs** [comment]
Also antags get thermals, hv ammo and crackers roundstart from uplink so we should place some some for IH in armory? I'm not against vendor, it's a good idea.

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1143,6 +1143,52 @@
 	idle_power_usage = 211 //refrigerator - believe it or not, this is actually the average power consumption of a refrigerated vending machine according to NRCan.
 	auto_price = FALSE
 
+/obj/machinery/vending/medical/commercial
+	name = "MiniPharma Trade"
+	desc = "Medical drug dispenser."
+	icon_state = "med"
+	icon_deny = "med-deny"
+	req_access = list(access_medical_equip)
```
Comment (Voldirs) [inline code/game/machinery/vending.dm:1151]: Also why?

**Comment by TheShown911** [comment]
Hmmm, maybe they should work for those painkillers and chems instead of digging trash for the first 30 minutes of the round?

**Comment by SirRichardFrancis** [comment]
> Also antags get thermals, hv ammo and crackers roundstart from uplink so we should place some some for IH in armory? I'm not against vendor, it's a good idea.

Also captain have two millions credits roundstart, while vagas don't. It's roleplaying game (ha ha) and players not start equal, but according to their character's role. My point is that military organization not having basic chems supply is silly.

> Hmmm, maybe they should work for those painkillers and chems instead of digging trash for the first 30 minutes of the round?

Like?

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (Voldirs) [reply code/game/machinery/vending.dm:1151]: Why have you put medical equipment access requirement? So med spec has to hand out pills that they bought with their own money? I don't really get it.

Comment (SirRichardFrancis) [reply code/game/machinery/vending.dm:1151]: 👀

Comment (SirRichardFrancis) [inline code/game/machinery/vending.dm:1151]:
```suggestion
	req_access = list()
```

Comment (SirRichardFrancis) [reply code/game/machinery/vending.dm]: It shouldn't have been there and it's no longer is. Thanks for pointing out.

**Comment by FortuneMaster** [comment]
on one hand I'd have prefered if med stuff would be left to moemed, on the other it already leads to IH just coming in chemestry anyway without asking. So while this is a shitty solution, its a fast one since a good one would require update for both factions

**Comment by SirRichardFrancis** [comment]
> on one hand I'd have prefered if med stuff would be left to moemed, on the other it already leads to IH just coming in chemestry anyway without asking. So while this is a shitty solution, its a fast one since a good one would require update for both factions

And proper update will come sooner or later, but that's a lot of work and there is other priorities.
Also high pop would pretty much solve that issue with chemistry, but alas, we don't have it.

**Comment by Voldirs** [comment]
There is nothing more permanent than temporal fixes

**Comment by Humonitarian** [comment]
> Mixing tric from bottles is practical, but autoinjectors look like a thing military would use.

This is why I propose dylovene and inaprovaline autoinjectors only. Those can be mixed in the bloodstream, by injecting both at once.

> "Toxin and sleep toxin" is useless for healing, m'okay, and IHS have no actual need for those.

Sleep toxin is very much so useful at pacification. As MedSpec I always steal a bottle of it from Moebious Medical.

> I've been thinking about giving them personal medkits, with like couple of autoinjectors and salve/bandages, but that will raise even more complains, so whatever.

Actually fun idea, don't throw it away! I think IH OP personal closets having a personal aid kit with a few autoinjectors and basic salves/bandages would be pretty fun!

**Comment by FortuneMaster** [comment]
> There is nothing more permanent than temporal fixes

yeah, but since TM update is on it's way, we'll eventually get an IH one too

**Comment by FortuneMaster** [comment]
> > And proper update will come sooner or later, but that's a lot of work and there is other priorities. 
yep, that's pracrically the only reason I tolerate this PR.
> > Also high pop would pretty much solve that issue with chemistry, but alas, we don't have it.
P.S. - from my experience it doesn't and they still just come in no matter if moemed is present or not.

**Comment by Rumplenutskin** [comment]
> 
> How medspecs going on a pilgrimage to chemistry roundstart is almost comical, even more so IH making a ghetto chemistry, a real bruh moment. That _if_ they concerned about not dying the moment gunfight starts. They mostly don't and one ~powergamer~ skilled antagonist slaughters entire department no problem. As if moe actually care to make and sell chems. Most often than not there is nobody to man chemistry.

It's always wild to me the differences in players experiences. There's some rounds I can't give combat drugs away for free to IH.  From an MBO main's perspective the system is working as it should. 

Medspecs need to visit moebius round start? That's good, we want faction interaction. When I've played with competent med specs they usually show up within the first 10-20 minutes before anything actually happens, and pay or barter for their meds. Sometimes I take credits sometimes I ask them to return the favor later. It forms a small bond for that round that wouldn't have happened otherwise.

Moemed is a mostly reactive job and services rendered are mostly free. Selling drugs to IH gives me something to do while I'm twiddling my thumbs. I can make a couple credits and feel like a true capatalist. IH is my main buyers, the rest of the crew only ever wants space drugs. Giving IH a chem disp just feels like the least inspired fix.

**Comment by TheShown911** [comment]
The problem is that there is not enough interaction with players, it happens <10% of the time right? So the solution? Make it happen 0% of the time, hmm yes good idea. I do not approve of this PR.

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Please remove the vendor because this is going to fuck with medbay even more given IH don't even have to ~~break into~~ buy chems from chemistry and encourages departments not interacting with each other.
Get your fix from a qualified chemist, not a soulless machine.

**Comment by Kegdo** [comment]
Considering that there is now a way to directly heal limbs, I think that this pr/map changes should be remade

**Comment by Kurgis** [comment]
_Just replace the Brig's Medbay with a featureless room with two-by-three floor space containing a WallMed prisoners need to purchase from._

**Comment by Kurgis** [comment]
In more seriousness, from the very moment the Brig's Little Medical Ward was added there has been problems with it which this does attempt to redress. Besides the changes of the Specialist's Office from a re-purposed closet to something which at least attempts to plaster itself as professional looking, I like the lay-out changes made to the ward, as they've been asked after for quite some time until people finally got used to the bad lay-out as "normal".

**Comment by Kurgis** [comment]
> The problem is that there is not enough interaction with players, it happens <10% of the time right? So the solution? Make it happen 0% of the time, hmm yes good idea. I do not approve of this PR.

> Please remove the vendor because this is going to fuck with medbay even more given IH don't even have to break into buy chems from chemistry and encourages departments not interacting with each other.
Get your fix from a qualified chemist, not a soulless machine.

I find myself in rare disagreement with the both, mainly with the very basis of the argument, that this shouldn't be done due to limiting player interactions. The Medical Specialist by its very role limits inter-departmental interaction, and therefore more dynamic player interaction in turn. If I wanted to increase player interaction I'd remove the Medical Specialist as a role completely, delegating their duties unto the Warden's shoulders instead. Now Operatives will need to deal with the Moebius department for essential duties rather than their (semi-)reliable assistants.

**Pull request merged**